modules/editable-layers/src/utils/geojson-types.ts

```diff
@@ -20,36 +39,6 @@ export type AnyCoordinates =
   | MultiLineStringCoordinates
   | MultiPolygonCoordinates;
 
-export type Point = {
-  type: 'Point';
-  coordinates: PointCoordinates;
-};
-
-export type LineString = {
-  type: 'LineString';
-  coordinates: LineStringCoordinates;
-};
-
-export type Polygon = {
-  type: 'Polygon';
-  coordinates: PolygonCoordinates;
-};
-
-export type MultiPoint = {
-  type: 'MultiPoint';
-  coordinates: MultiPointCoordinates;
-};
-
-export type MultiLineString = {
-  type: 'MultiLineString';
-  coordinates: MultiLineStringCoordinates;
-};
-
-export type MultiPolygon = {
-  type: 'MultiPolygon';
-  coordinates: MultiPolygonCoordinates;
-};
-
 export type Geometry = Point | LineString | Polygon | MultiPoint | MultiLineString | MultiPolygon;
```

Comment:
- Please add a `/** */` TSDoc style comment for each new definition. 

Unrelated to this PR
- I am not sure I like the `Geometry` and `Polygonal` types as these are not following normal conventions. But changing them probably requires updating other code, so would be for a different PR.
- For one thing, `Geometry` is typically called `Feature`

Reply:
This is your code. I only import the types of geojson.